annotator-core/src/test/java/edu/ucr/cs/riple/core/tools/TReport.java

```diff
@@ -34,7 +34,11 @@
 import java.util.stream.Collectors;
 import javax.annotation.Nullable;
 
-/** Wrapper class for {@link Report} with default values, used to create tests. */
+/**
+ * Wrapper class for {@link Report} used to create dummy reports (with default values) which in turn
```

Comment:
```suggestion
 * Wrapper class for {@link Report} used to create dummy reports (with default values) which
```

No need for this "in turn" here, since the reports are just directly used by test to [...] :)